test/component/will/testWillCodicilsNumber.js

```diff
@@ -1,9 +1,9 @@
-const TestWrapper = require('test/util/TestWrapper');
-const CodicilsDate = require('app/steps/ui/will/codicilsdate/index');
+const TestWrapper = require('test/util/TestWrapper'),
+    IhtCompleted = require('app/steps/ui/iht/completed/index');
```

Comment:
Please change this back to how it was being declared as a constant on each line rather than multiple declaration